.github/workflows/e2e-test.yaml

```diff
@@ -51,13 +51,13 @@ jobs:
           set -o pipefail
           # this is required to use aws-iam-authenticator.
           export PATH=$PATH:$HOME/bin
-          cp ../variables.tf ../pkg/modules/AWS/asg_ec2_tagging/
-          echo "environment_name = \"dummy-environment\"" >> ../pkg/modules/AWS/asg_ec2_tagging/dummy-config.auto.tfvars
-          echo " locals {" > ../pkg/modules/AWS/asg_ec2_tagging/dummy-tfstate-local.tf
-          echo "     dynamodb_name = \"dummy\"" >> ../pkg/modules/AWS/asg_ec2_tagging/dummy-tfstate-local.tf
-          echo "     bucket_name   = \"dummy\"" >> ../pkg/modules/AWS/asg_ec2_tagging/dummy-tfstate-local.tf
-          echo "     bucket_key    = \"dummy\"" >> ../pkg/modules/AWS/asg_ec2_tagging/dummy-tfstate-local.tf
-          echo " }" >> ../pkg/modules/AWS/asg_ec2_tagging/dummy-tfstate-local.tf
+          cp ../variables.tf ../modules/AWS/asg_ec2_tagging/
```

Comment:
There shouldn't be any reference to our `/pkg` folder in the project anymore. I have tested installing and uninstalling with the new structure.